codegen/src/main/kotlin/org/jetbrains/bsp/generators/ir/SmithyToIr.kt

```diff
@@ -222,7 +226,9 @@ class SmithyToIr(val model: Model) {
             val hints = getHints(shape)
             val type = shape.accept(toTypeVisitor)
 
-            return listOfNotNull(type?.let { Def.Alias(shape.id, it, hints) })
+            return listOfNotNull(type?.let {
+                Def.Alias(shape.id, Type(ShapeId.fromParts("alias", it.shapeId.name), it.type), hints)
```

Comment:
fromParts nie bierze namespace'a jako pierwszy argument?

Reply:
bierze, po prostu chciałam dać jakiś inny namespace, żeby ułatwić renderowanie, ale może spróbuje jakoś inaczej to obejść

Reply:
poprawione